HelpSource/Classes/UnitTest.schelp

```diff
@@ -107,63 +107,92 @@ YourClass.test
 ::
 Runs the test class for code::YourClass::, which is assumed to be named code::TestYourClass::.
 
-If no test class if found it politely declines.
+If no test class is found, it politely declines.
 
 note::
-UnitTests for the Common library classes are kept in the CommonTests quark. 	This enables you to easily install and uninstall these tests.
+UnitTests for the Common library classes are kept in the CommonTests quark. This enables you to easily install and uninstall these tests.
 ::
 
 
 Make sure to implement the methods code::setUp:: and code::tearDown::. They will be called before and after the test.
 
 METHOD:: assert
 
+Asserts that an expression must be true.
+
 argument:: test
-Make sure that this returns a boolean, where code::true:: means that the test is passed.
+An expression which is expected to return a boolean, where code::true:: means that the test is passed.
```

Comment:
The clearest version of this is probably simply "a boolean, where...". Strictly speaking expressions don't return values, and the wording is inconsistent with how we describe other arguments. e.g. the description of message below isn't "an expression which is expected to return a string...".